src/pages/getting-started/app-guides/onboarding/index.mdx

```diff
@@ -2,22 +2,21 @@
 weight: 90
 category: "getting-started"
 subcategory: "app-guides"
-title: "Onboarding & Settings"
+title: "Onboarding"
```

Comment:
I have changed the name, so I also needed to change everything linking to it. Luckily, there were only 3 things - two are already changed as they were in this PR, and one more I will change when introducing changes to the other PR as there is a file I am working with, and I would like to avoid conflicts.